### ISSUE TITLE
Unable to install pywin32 with Python 3.8 and 3.10

### ISSUE THREAD

**Issue by CDAGaming** [mhammond/pywin32#1765]
Both Python 3.8 and 3.10 with PIP 20.0.2 and 21.2.4 show the following error:
```
ERROR: Could not find a version that satisfies the requirement pywin32 (from versions: none)
ERROR: No matching distribution found for pywin32```

Previously this package was able to install just fine, so I'm quite unsure if somethings changed recently.

**Comment by CDAGaming** [comment]
Further looking leads me to https://github.com/pypa/pip/issues/10545 showing the same symptoms here.

**Comment by cltj** [comment]
I'm getting the same issue.

**Comment by mauvealerts** [comment]
The [output](https://github.com/mhammond/pywin32/files/7312819/pip_download_pywin32_301.txt) of `pip  -v -v -v download pywin32==301` (with pip 21.2.4 and Python 3.10.0 on amd64) suggests that PyPi doesn't have a wheel tagged for 3.10.

I have no idea what could be wrong with 3.8, nor how 3.10 would have succeeded.

**Comment by qwarkys** [comment]
https://pypi.org/project/pywin32/ only lists python versions 3.5 to 3.9, which may account for pip failing for 3.10.  I have no idea about the 3.8 issue. I could install pywin32 on 3.9 with pip.

Don't files get uploaded to pypi by a maintainer, rather than some other process?

**Comment by mauvealerts** [comment]
pypiwin32 was last updated in 2018

**Comment by fsecada01** [comment]
Can the latest build be uploaded to pypi, then?  I'd rather not have to pin a git address to my requirements for my ongoing projects.

**Comment by mhammond** [comment]
I've no idea what the 3.8 failure is about, but build 302 has been released with support for 3.10.
